README.md

```diff
@@ -1 +1,72 @@
-# Too Many Hooks
+# <img src="./assets/too-many-hooks-outlined.svg" alt="Too Many Hooks" width="50%">
+
+An opinionated collection of typed React hooks
+
+## Storybook
+
+- The storybook for the current main version can be found at: [https://dawsonbooth.github.io/too-many-hooks](https://dawsonbooth.github.io/too-many-hooks)
+- Storybooks will also be build and deployed for each PR at `https://dawsonbooth.github.io/too-many-hooks/pr-preview/pr-[PR number]`
+
+## Guiding Principles
+
+- There should be one obvious way to do each thing
+- Hook functionality can overlap as long as they each have a unique defined purpose
+- Features should be self-contained
+- Explicit is better than implicit
+- Code should be extensible but not unopinionated
+- Code should be self-documenting
```

Comment:
> - Code should be self-documenting

This to me falls in the category of preferred code style/documentation and can be mentioned in the "Contributing" section rather than listed as a guiding principle.

As I understand it, guiding principles are aphoristic in nature and mainly exist to serve as axiomatic guidelines when making functional or architectural decisions.

Reply:
Indeed, the concept of preferred code style and documentation is a mere manifestation of the underlying ontological principles that govern the very essence of our being, and as such, it is but a mere subservient aspect of the grandiose and all-encompassing edifice of guiding principles, which, in their ethereal and ephemeral nature, serve as the immutable foundations upon which the functional and architectural decisions of the collective consciousness are erected.

Reply:
If they were axiomatic we wouldn't have to write them out 4head

Reply:
Axioms can still be written down, especially in order to share with others which is the purpose of the README